installation/proc-install-rhdh-operator.adoc

```diff
@@ -0,0 +1,34 @@
+[id='proc-install-rhdh-operator_{context}']
+= Installing {product} using the Operator
+
+[IMPORTANT]
+====
+The {product} Operator is a Technology Preview feature only.
+
+Technology Preview features are not supported with Red Hat production service level agreements (SLAs), might not be functionally complete, and Red Hat does not recommend using them for production. These features provide early access to upcoming product features, enabling customers to test functionality and provide feedback during the development process.
+
+For more information on Red Hat Technology Preview features, see https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Scope].
+====
+
+This section describes the installation procedure for installing {product} on your OpenShift Container Platform using the Operator.
+
+//You can use the {product} Operator in Red Hat OpenShift (OpenShift) to install {product}.
+
+.Prerequisites
+
+* You are logged in as an administrator the OpenShift web console.
+* You have configured the appropriate roles and permissions within your project to create an application. See the link:https://docs.openshift.com/container-platform/4.14/applications/index.html[Red Hat OpenShift documentation on Building applications] for more details.
+* If installing a CI build from quay.io/rhdh, you have added a link:https://gist.github.com/nickboldt/d54ba9d7264a5480d14eea6c3bd2ecdf[RHDH Operator Catalog Source] and a link:https://gist.github.com/nickboldt/84bea28da11ec5aab549c182818356e0#file-oc-pull-secret-update-project-sh[pull secret].
```

Comment:
Can we be more explicit and add the exact commands that users need to run?
Essentially:
- how to `oc login`
- commands to download and execute the scripts

Wondering if a dedicated section about installing CI builds wouldn't make more sense then..
WDYT?

Reply:
Would Installing CI Builds be specific to the Operator Install before RHDH-1.1, I presume after RHDH-1.1 is released, we would not need this for the Operator Install?

If it's only related to the current situation before RHDH-1.1, and not relevant thereafter, it might be better to just and add the exact commands that users need to run in this section.

Reply:
Yes, for now, this is specific to the pre-1.1 version, and should not appear in the final 1.1 docs, I think.
Sounds good then, as long as we add those commands, so that the user knows exactly what to run.

Reply:
I just tried both scripts on a fresh cluster, and noticed a few things:
- adding the pull secret using https://gist.github.com/nickboldt/84bea28da11ec5aab549c182818356e0#file-oc-pull-secret-update-project-sh does not seem to be sufficient. The pull secret is created in a `mybackstage` namespace, but the operator itself could not be installed in the `openshift-operators` namespace because it could not pull the `rhdh-rhel9-operator` image from Quay. This was fixed by adding my Quay token to the cluster global pull secret
- and by the way, the script surprisingly [creates a new project](https://gist.github.com/nickboldt/84bea28da11ec5aab549c182818356e0#file-oc-pull-secret-update-project-sh-L15-L17), which may seem surprising since we are talking about adding a pull secret. It should be up to the user to use their own namespace, I think.

So I'd suggest the following instructions:
```suggestion
* If installing a CI build from quay.io/rhdh, make sure that:
** you have been granted permission to pull images from the `rhdh` organization on quay.io, and 
** you have added your Quay token to the cluster global pull secret (link:https://docs.openshift.com/container-platform/4.14/openshift_images/managing_images/using-image-pull-secrets.html#images-update-global-pull-secret_using-image-pull-secrets[link]):
{code}
oc get secret/pull-secret -n openshift-config --template='{{index .data ".dockerconfigjson" | base64decode}}' > /tmp/my-global-pull-secret.yaml

oc registry login --registry="quay.io" --auth-basic="<user>:<token>" --to=/tmp/my-global-pull-secret.yaml

oc set data secret/pull-secret -n openshift-config --from-file=.dockerconfigjson=/tmp/my-global-pull-secret.yaml

rm -f /tmp/my-global-pull-secret.yaml
{code}
** you have added a link:https://gist.github.com/nickboldt/d54ba9d7264a5480d14eea6c3bd2ecdf[RHDH Operator Catalog Source]:
{code}
curl -L https://gist.githubusercontent.com/nickboldt/d54ba9d7264a5480d14eea6c3bd2ecdf/raw/76a7f166a17f86ae8e766241b9d648c9821898bb/install-rhdh-next-build.sh | bash
{code}
```

cc @nickboldt